src/commons/IconButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ComponentPropsWithoutRef, FC } from "react";
+
+interface IconButtonProps extends ComponentPropsWithoutRef<"button"> {
+  icon: JSX.Element | JSX.Element[];
+}
+
+const IconButton: FC<IconButtonProps> = ({
+  icon,
+  disabled,
+  className,
+  ...iconButtonProps
+}) => {
+  return (
+    <button
+      className={`flex items-center justify-center h-11 w-11 rounded-full text-whiteText shadow-md transition-colors ${
+        disabled
+          ? "bg-disableButton cursor-default"
+          : "bg-primaryBlue hover:bg-hoverBlue active:bg-activeBlue"
+      } ${className || ""}`}
+      {...iconButtonProps}
+    >
+      <div>{icon}</div>
```

Comment:
Creo que acá no haría falta ponerle el <div/>.

Pero en caso que sea necesario poner un contenedor (sólo sí es necesario), debería ser un <span/> porque los button sólo deberían recibir elementos en línea, no en bloque.

[https://stackoverflow.com/questions/12982269/is-it-semantically-incorrect-to-put-a-div-or-span-inside-of-a-button](url)

Reply:
Ya modifique la linea